ceres.py

```diff
@@ -819,6 +882,57 @@ class SliceDeleted(Exception):
   pass
 
 
+def aggregate(aggregationMethod, values):
+  # Filter out None values
+  knownValues = filter(lambda x: x is not None, values)
+  if len(knownValues) is 0:
+    return None
+  # Aggregate based on method
+  if aggregationMethod == 'average':
+    return float(sum(knownValues)) / float(len(knownValues))
+  elif aggregationMethod == 'sum':
+    return float(sum(knownValues))
+  elif aggregationMethod == 'last':
+    return knownValues[len(knownValues) - 1]
```

Comment:
Ah, coming from another language I'm used to writing ``[$ - 1]`` and immediately think of ``[-1]`` as an out of bounds index.